.github/workflows/ci-extends.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+name: ci-extends
+
+on:
+  workflow_dispatch:
+    inputs:
+      saucelab_test:
+        description: 'Enable run a Sauce lab test'
+        default: true
+        required: false
+        type: boolean
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+
+# Limit the access of the generated GITHUB_TOKEN
+permissions:
+  contents: read
+  statuses: write
```

Comment:
What's the reason for this write access?

Reply:
I removed the unneeded permission.